Adding New Miscellaneous Items

Eryneril opened the pull request.

Eryneril committed:
Update 07_Items.txt

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+Industrial Paint (35$):
+	A metal canister of wear-resistant paint. Can be used 
+with a brush or similar implement to cover up to 20 square meters of
+flooring, wall or other surface. Paint comes in the following colors.
```
Non-block: End this line with a colon?

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+solution in either an area or a thin line. As the solution dries, 
+it expands and hardens to accomodate the space given. Can be used
+to insulate an area, temporarily patch up a damaged wall, 
+seal ducts, etc. One can has 20 square meter's worth of usage.
```
n-b: "One can contains" and "worth of usage" could be written "worth of foam"
block: Cubic meters may be a more useful unit of measurement.

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in an area. When this solution comes into contact with
+adhesive foam or similar substances, they will dissolve into a vapor.
+One can has 20 square meter's worth of usage.
```
"worth of solvent"
Idk if cubic or square meters is more useful here.

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A metal crate secured with an electronic keypad. Can hold
+up to 150lbs. The keypad can be opened with either the correct numeric
+password, a DC 85 Strength skill check or with a DC 75 hacking 
+skill check.
```
We may want to know volume or the crate dimensions too.
n-b: Can haz metric system pls?

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A reinforced storage crate designed to carry heavy cargo. Can
+hold up to 600lbs. The keypad can be opened with either the correct 
+numeric password, a DC 105 Strength skill check or with a DC 95 hacking 
+skill check.
```
Volume/dimensions and metric :)

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A shovel built with a compacted ultrasonic oscillation
+generator in the handle. When activated, the generator causes the
+blade of the shovel to vibrate at a fast speed. This increases
+the rate it can excavate material by a considerable degree. 
```
For fun, be specific about how much faster the vibroshovel is than a regular shovel (It can be totally random to start; we can tweak it later).

rosebrallier reviewed (changes requested):
This is awesome! Thanks Will; we need these so much!
One general note, swap the position of the cost and the "$" sign (I'm our friendly neighborhood syntax fusser).

Eryneril committed:
Update 07_Items.txt

Eryneril committed:
Update 07_Items.txt

Turtlelord26 commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+solution in either an area or a thin line. As the solution dries, 
+it expands and hardens to accomodate the space given. Can be used
+to insulate an area, temporarily patch up a damaged wall, 
+seal ducts, etc. One can has 20 cubic meter's worth of foam.
```
20 cu m is a lot of foam. Maybe just 1 cubic meter per can? Otherwise it's an enormous can.

bleehu replied:
Adhesive foam is super cool! Great idea!

Turtlelord26 commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in an area. When this solution comes into contact with
+adhesive foam or similar substances, they will dissolve into a vapor.
+One can has 20 cubic meter's worth of solvent.
```
Same as for the adhesive, 20 cu m is probably too much.

bleehu replied:
I mean, the solvent is probably compressed, so it would expand to fill a much larger volume than the can it came out of... but still, 20 m^3 is a _lot_.

Turtlelord26 commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+One can has 20 cubic meter's worth of solvent.
+
+Basic Storage Crate ($125):
+	A 1mx0.5mx0.5m metal crate secured with an electronic keypad. 
```
This seems small. Can we make it a full 1x1x1m cube?

bleehu replied:
Crates! Its so simple and fantastic! I love it!

Turtlelord26 commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+	A shovel built with a compacted ultrasonic oscillation
+generator in the handle. When activated, the generator causes the
+blade of the shovel to vibrate at a fast speed. This decreases
+the time it can take to excavate material by half the normal rate. 
```
^to^ half the normal rate.

Turtlelord26 reviewed (changes requested):
👍 🐢 👍

bleehu commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+operator can see out of with a display set into the controller.
+The controller can transmit its signal to the drone over a 
+maximum range of 600m before the signal is lost. A DC 70 
+spotting skill check can identify it for what it truly is.
```
non-blocking: This is fantastic! Can we add a hacking DC and effect for hackers?

bleehu commented on Items/07_Items.txt:
```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
+
+Shovel ($90):
+	A hand tool with a broad blade typically used for digging, 
+lifting and moving bulk material such as soil, ore, snow and sand.
```
Maybe a bit too pricey for a good shovel? Does it collapse for ease of packing?

bleehu reviewed (changes requested):
Some minor changes, but this is super fantastic!!!

bleehu commented:
Once handled, this will close out #96

Eryneril committed:
Updated Foam, Solvent, Crate, F. Fauna and Shovel

Changed the Foam and Solvent to hold 1 cubic meter each, increased the size of the Basic Crate, gave a Hacking skill effect to the False Fauna and reduced the price of the Shovel and the Vibroshovel.

bleehu reviewed (approved):
:)

Eryneril committed:
Updated False Fauna Hacking Checks

Eryneril committed:
False Fauna Typo Resolution
